samples/databricks-cicd/README.md

```diff
@@ -0,0 +1,70 @@
+# Overview
+
+__What does this sample demonstrate:__
+
+This sample demonstrates how to run unit tests on Python code using [Databricks Connect](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/databricks-connect), run integration tests on Databricks notebooks, and deploy a Databricks job, all in [GitHub Actions](https://docs.github.com/en/actions/learn-github-actions).
+
+This sample contains a single notebook [main_notebook.py](src/main_notebook.py) which calls [some_function](src/dbcicdlib/some_func.py) in a Python module. We run [unit test on some_function](test/unittests/test_some_func.py) and [integration test on main_notebook.py](test/run_notebook_tests.sh).
+
+This sample also shows how to [update the job if a job with the same name already exists](../../.github/workflows/databricks-cicd.yml#L136). Without this, if the CD pipeline publishes a job, it's a new job with a new job id even if another job with the same name already exists.
+
+Another small feature used in this sample is to use a [pre-commit config hook](.pre-commit-config.yaml) to run [detect-secrets](https://github.com/Yelp/detect-secrets) to prevent secrets from being checked into code.
```

Comment:
This is very nice cool feature! It comes later in the readme but maybe worth to point out that the yaml description comes from this project `https://pre-commit.com/` I was confused because I learned that git hooks are mainly bash scripts, but that tool looks very handy. At least if you are in a python environment.

Reply:
Added a link to pre-commit.com here.